src/the_tale/the_tale/settings.py

```diff
@@ -18,7 +18,7 @@
 META_CONFIG_FILE = os.path.join(PROJECT_DIR, 'meta_config.json')
 META_CONFIG = dext_meta_config.MetaConfig(config_path=META_CONFIG_FILE)
 
-DEBUG = False
+DEBUG = True
```

Comment:
```suggestion
DEBUG = False
```
По-умолчанию в проекте используем продовские конфиги, чтобы уменьшить риск попадания на прод того, чего там быть не должно